templates/docs/examples/patterns/tabs/tab-buttons.html

```diff
@@ -0,0 +1,97 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Tabs / Tab Buttons{% endblock %}
+
+{% block standalone_css %}patterns_tab-buttons{% endblock %}
+
+{% block content %}
+
+<div class="p-tab-buttons">
+  <div class="p-tab-buttons__list" role="tablist" aria-label="Juju technology">
+    <button
+      class="p-tab-buttons__button has-icon"
+      role="tab" 
+      aria-selected="true" 
+      aria-controls="olm-tab" 
+      id="olm"
+    >
+      <i class="p-icon--information"></i><span>Charmed Operator Lifecycle Manager</span>
```

Comment:
The spec suggests that you should only use icons if you can put icons on all buttons. I think we should make our examples also consistent with this recommendation.

So probably we need a separate example with icons on all buttons?